UtopiaCity/Services/Reservation/ReservationService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using UtopiaCity.Data;
+
+namespace UtopiaCity.Services.Reservation
+{
+    public class ReservationService
+    {
+        private readonly ApplicationDbContext _dbContext;
+
+        public ReservationService(ApplicationDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+        public async Task<Models.PublicCatering.Reservation> GetReservationById(string id)
+        {
+            return await _dbContext.Reservations.Include(r =>r.Restaurant).FirstOrDefaultAsync(x => x.Id.Equals(id));
+        }
+        public async Task<List<Models.PublicCatering.Reservation>> GetReservations()
```

Comment:
will look better with using Models.PublicCatering, will allow to simplify names

Reply:
Yes, but it doesn't work. I don't know why.